tests/rptest/services/redpanda.py

```diff
@@ -1119,6 +1172,41 @@ def write_node_conf_file(self, node, override_cfg_params=None):
                 doc["redpanda"].update(override_cfg_params)
             conf = yaml.dump(doc)
 
+        if self._security.tls_provider:
```

Comment:
write_node_conf_file is called on each node startup: it would be more realistic to write the certificates out on first startup only -- there's a path in start() where write_bootstrap_cluster_config is called on first start: maybe put the TLS cert creation into a separate function and call it from the same place?

(The actual update of the node config file is fine in this function, it's just the creation of the certs I'm suggesting should be a onetime thing)

Reply:
thanks I missed this detail of how startup works. fixed.